### PR TITLE
make DownloadDirectory returns outputs

### PR DESCRIPTION
**Pull request by atetubou**
We need the list of downloaded files in our client.

### CODE REVIEW

**Comment by atetubou** [comment]
@philwo do you think 'go: not found' error is related to recent your work in https://buildkite.com/bazel/remote-apis-sdks/builds/931#702a6e1d-78d0-43d2-a60f-fd17934b380d ?

**Comment by philwo** [comment]
Sorry, yes 🤦🏻‍♂️ @fweikert will fix it ASAP. I pushed the Docker containers but forgot to promote the VM images to production.

**Comment by philwo** [comment]
Fixed!

**Comment by atetubou** [comment]
@philwo I saw other error now.
And it seems not specific to this PR?

File: go/pkg/client/cas_test.go [hunk 1]
```diff
@@ -883,13 +883,9 @@ func TestDownloadDirectory(t *testing.T) {
...
 	d := digest.TestNewFromMessage(dir)
-	execRoot, err := ioutil.TempDir("", "DownloadDirectory")
-	if err != nil {
-		t.Fatalf("failed to make temp dir: %v", err)
-	}
-	defer os.RemoveAll(execRoot)
+	execRoot := t.TempDir()
 
-	err = c.DownloadDirectory(ctx, d, execRoot, cache)
+	_, err = c.DownloadDirectory(ctx, d, execRoot, cache)
```
Comment (ola-rozenfeld) [inline go/pkg/client/cas_test.go:888]: nit: If you're returning them, might as well test that they are expected.

**Comment by philwo** [comment]
> @philwo I saw other error now.
> And it seems not specific to this PR?

Yes, unfortunately the new Buildkite Agent that was rolled out with the new VMs had a bug 😭 I had to roll it back to an older version. It should be fine now.

Your PR test passed now: https://buildkite.com/bazel/remote-apis-sdks/builds/934#_

File: go/pkg/client/cas_test.go (hunk 1 above)
Comment (atetubou) [reply go/pkg/client/cas_test.go]: updated.

**Comment by atetubou** [comment]
@philwo thank you for the fix, and I can use go1.15 now.

**Comment by atetubou** [comment]
Please merge if this looks good.

**Review by ola-rozenfeld — APPROVED** [review]

**Pull request merged**